grafana_client/knowledge.py

```diff
@@ -285,7 +285,35 @@ def query_factory(datasource, model: Optional[dict] = None, expression: Optional
                 "default": None,
             },
         ]
+    elif datasource_type == "grafana-redshift-datasource":
+        query = {
+            "datasource": {
+                "type": datasource["type"],
+                "uid": datasource.get("uid"),
+            },
+            "datasourceId": datasource.get("id"),
+            "rawSQL": expression,
+        }
 
+        attrs = [
+            {
+                "name": "format",
+                "default": 0,
+                "choices": [0, 1],
+            },
+            {
+                "name": "intervalMs",
+                "default": 30000,
+            },
+            {
+                "name": "maxDataPoints",
+                "default": 1441,
+            },
```

Comment:
Hi again. Is this a sane default value? Or should this spot be adjusted in one way or another before merging?

/cc @peekjef72